bundles/org.openhab.binding.linky/README.md

```diff
@@ -1,66 +1,1049 @@
+  |----------------------------------------------------------------|-------------------|-------------------------------------------------------------------------------|
+  | linky-remote-yearly#thisYear                                   | consumption       | Current year energy usage                                                     |
+  | linky-remote-yearly#lastYear                                   | consumption       | Last year energy usage                                                        |
+  | linky-remote-yearly#year-2                                     | consumption       | year-2 energy usage                                                           |
+  | linky-remote-yearly#consumption                                | consumption       | timeseries for years energy usage                                             |
+  | linky-remote-yearly#maxPower                                   | power             | timeseries for max-power yearly usage                                         |
+ 
+- The linky-remote-load-curve group will give you access to load curve data with granularity as low as 30mn
+
+  | Channel ID                                                     | Item Type         | Description                                                                   |
+  |----------------------------------------------------------------|-------------------|-------------------------------------------------------------------------------|
+  | linky-remote-load-curve#power                                  | power             | The load curve data                                                           |
+
+### Full Example
+
+#### Remote Enedis Web Connection
+
+```java
+Bridge linky:enedis-web:local "EnedisWebBridge" [ username="example@domaine.fr", password="******", internalAuthId="******" ]
+
+Thing linky:linky:linkyremotexxxx "Linky Remote xxxx" (linky:enedis-web:local) [ prmId="xxxx" ]
+```
+
+```java
+Number:Energy ConsoHier "Conso hier [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:daily#yesterday" }
+Number:Energy ConsoSemaineEnCours "Conso cette semaine [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:weekly#thisWeek" }
+Number:Energy ConsoSemaineDerniere "Conso semaine dernière [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:weekly#lastWeek" }
+Number:Energy ConsoMoisEnCours "Conso ce mois [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:monthly#thisMonth" }
+Number:Energy ConsoMoisDernier "Conso mois dernier [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:monthly#lastMonth" }
+Number:Energy ConsoAnneeEnCours "Conso cette année [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:yearly#thisYear" }
+Number:Energy ConsoAnneeDerniere "Conso année dernière [%.0f %unit%]" <energy> { channel="linky:enedis-web:local:yearly#lastYear" }
```

Comment:
Is it an error ? It looks like thing linky:enedis-web:local has no channels. Should linky:linky:linkyremotexxxx be rather used ?

Reply:
Note that group name was not updated in the example.
But look at my other comment, I would prefer the historical group names to avoid breaking all channel links.